hivemind/p2p/p2p_daemon_bindings/datastructures.py

```diff
@@ -128,6 +129,12 @@ def from_protobuf(cls, peer_info_pb: p2pd_pb2.PeerInfo) -> "PeerInfo":
         addrs = [Multiaddr(addr) for addr in peer_info_pb.addrs]
         return PeerInfo(peer_id, addrs)
 
+    @classmethod
+    def from_endpoint(cls, endpoint: Endpoint) -> "PeerInfo":
+        peer_id = PeerID.from_base58(endpoint[0])
+        addrs = [Multiaddr(a) for a in endpoint[1]]
+        return PeerInfo(peer_id, addrs)
```

Comment:
Eventually yes. I renamed it into `from_tuple` since `Endpoint` is removed. This method needed in call `moe/server/dht_handler/get_experts` to convert dht values, which are sotred as tuples of strings into `PeerInfo`.

Reply:
I think we would be able to remove this method in the next PR that will get rid of sending multiaddrs. No need to worry about it in this PR.